### ISSUE TITLE
Question about hyper parameters

### ISSUE THREAD

**Issue by Dinghow** [AnTao97/dgcnn.pytorch#3]
Thanks for your pytorch implementation of dgcnn, it's a helpful job. BTW, I wonder how did you set the hyper parameters of classification task on ModelNet40. I used default setting, but couldn't reach the performance shown in README, I just got 92.7 overall accuracy and 89.2 mean class accuracy,  with other adjustment, it helped a little. I would be glad if you can tell us how to set the parameters, thanks a lot!

**Comment by antao97** [comment]
I have already written 

"**Tip:** The result of point cloud experiment usually faces greater randomness than 2D image. We suggest you run your experiment more than one time and select the best result." 

in README. This tip does take effect.

Another thing that might influence the performance is batch size. The batch size I used is suited for 4 gpu cards (2080 ti). So if you don't have 4 gpu cards and thus can't use the default batch size, the performance may be worse than mine.

**Comment by Dinghow** [comment]
> I have already written
> 
> "**Tip:** The result of point cloud experiment usually faces greater randomness than 2D image. We suggest you run your experiment more than one time and select the best result."
> 
> in README. This tip does take effect.
> 
> Another thing that might influence the performance is batch size. The batch size I used is suited for 4 gpu cards (2080 ti). So if you don't have 4 gpu cards and thus can't use the default batch size, the performance may be worse than mine.

Thanks for your prompt reply. I've tried 4~5 times and couldn't get a better performance, and I set the batch_size with 32 or 64 on two Tesla V100. Emmm, I'm testing my ideas based on your repo, it has a little improvement than the baseline I tested on my own platform, but not better than the best result you got (lol), thus it bothers me, I would try a few more times, thanks for your implementation again!

**Comment by antao97** [comment]
I indeed get my result in README using the code in this repo without any adjustment. Hope you get my result after a few more runs!

**Comment by thematrixduo** [comment]
same here, the best accuracy I can get is 92.6.

Can I confirm that you used SGD optimizer with lr 0.1 and momentum 0.9 with Cosine Annealing?

**Comment by antao97** [comment]
> same here, the best accuracy I can get is 92.6.
> 
> Can I confirm that you used SGD optimizer with lr 0.1 and momentum 0.9 with Cosine Annealing?

Yes. I use the default settings to get my result.

**Comment by thematrixduo** [comment]
> > same here, the best accuracy I can get is 92.6.
> > Can I confirm that you used SGD optimizer with lr 0.1 and momentum 0.9 with Cosine Annealing?
> 
> Yes. I use the default settings to get my result.

What about batch size? The default setting has batch size 32, which probably will be too small if you distribute training across 4 GPUs.

**Comment by Dinghow** [comment]
> > > same here, the best accuracy I can get is 92.6.
> > > Can I confirm that you used SGD optimizer with lr 0.1 and momentum 0.9 with Cosine Annealing?
> > 
> > 
> > Yes. I use the default settings to get my result.
> 
> What about batch size? The default setting has batch size 32, which probably will be too small if you distribute training across 4 GPUs.

I set batch_size = 64 on two V100, still got 92.7，without sync_bn.

**Comment by antao97** [comment]
> > > same here, the best accuracy I can get is 92.6.
> > > Can I confirm that you used SGD optimizer with lr 0.1 and momentum 0.9 with Cosine Annealing?
> > 
> > 
> > Yes. I use the default settings to get my result.
> 
> What about batch size? The default setting has batch size 32, which probably will be too small if you distribute training across 4 GPUs.

The batch size is set as 32. What I'm saying is that if you run experiment with 2048 points, you indeed need more than 2 cards. So I said I use 4 cards.

Note that the DGCNN author chooses the best model tested on test set, not the final model in the last epoch. I follow this rule to choose model.

**Comment by imankgoyal** [comment]
Hello, In case you are still facing this issue, we do an analysis of this issues related to reproducing the modelnet40 results here: https://github.com/princeton-vl/SimpleView. You might find it useful.
